### PR TITLE
Project NorthStar Azure Cohort3

### PR DESCRIPTION
**Pull request by glennthomas1**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

Adding 21 new quickstarts/dashboards as part of the Cohort 3 entities of Project NorthStar Azure, from the Product Partnership team.

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [X] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/azure/azure-hdinsight/config.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  Azure HDInsight is a managed, full-spectrum, open-source analytics service in the cloud for enterprises. With HDInsight, you can use open-source frameworks such as, Apache Spark, Apache Hive, LLAP, Apache Kafka, Hadoop and more, in your Azure environment.
+  
+  ### New Relic Azure HDInsight quickstart features
+
+  A standard dashboard that tracks key indicators like categorized gateway requests, pending memory, pending CPU. It runs custom queries and visualizes the data immediately.
+
+  ### Why monitor Azure HDInsight with New Relic?
+
+  [New Relic Azure HDInsight](/microsoft-azure-integrations/azure-integrations-list/azure-hdinsight-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure HDInsight via different metrics including categorized gateway requests, pending memory and pending CPU.
```
Comment (josephgregoryii) [inline quickstarts/azure/azure-hdinsight/config.yml:13]: I think we might be missing the host of this link!

File: quickstarts/azure/azure-purview/config.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  Microsoft Purview provides a unified data governance solution to help manage and govern your on-premises, multicloud, and software as a service (SaaS) data. Easily create a holistic, up-to-date map of your data landscape with automated data discovery, sensitive data classification, and end-to-end data lineage. Enable data consumers to access valuable, trustworthy data management. 
+ 
+  ### New Relic Microsoft Purview quickstart features
+
+  A standard dashboard that tracks key indicators like scan cancelled count, scan completed count, scan failed count, data map capacity units count. It runs custom queries and visualizes the data immediately.
+
+  ### Why monitor Microsoft Purview with New Relic?
+
+  [New Relic Microsoft Purview](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-purview-monitoring-integration/) monitoring quickstart empowers you to track the performance of Microsoft Purview via different metrics including scan failed count, scan cancelled count, scan completed count, data map capacity units count.
```
Comment (josephgregoryii) [inline quickstarts/azure/azure-purview/config.yml:13]: Might be missing the host name here as well

**Review by josephgregoryii — COMMENTED** [review]
Looks great! couple areas to cleanup, but once those are addressed should be good to go

**Comment by josephgregoryii** [comment]
Also to note: There are some dashboards missing images in this PR as well

File: quickstarts/azure/azure-app-service-environment/config.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  App Service Environment represents the collection of physical resources for the App service. An App Service Environment can have multiple web apps. In other words, we can have multiple web apps in an App Service Environment.
+
+  ### New Relic Azure App Service Environment quickstart features
+
+  A standard dashboard that tracks key indicators like bytes received, bytes sent, cpu percentage, disk queue length, memory percentage. It runs custom queries and visualizes the data immediately.
+
+  ### Why monitor Azure App Service Environment with New Relic?
+
+  [New Relic Azure App Service Environment](//docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-app-service-environment-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure App Service Environment via different metrics including account  bytes received, bytes sent, cpu percentage, disk queue length, memory percentage.
```
Comment (seemantk) [inline quickstarts/azure/azure-app-service-environment/config.yml:13]:
```suggestion
  [New Relic Azure App Service Environment](//docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-app-service-environment-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure App Service Environment via different metrics including account bytes received, bytes sent, cpu percentage, disk queue length, memory percentage. 
```

File: quickstarts/azure/azure-data-explorer/config.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+  ### Why monitor Azure Data Explorer with New Relic?
+
+  [New Relic Azure Data Explorer](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-data-explorer-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure Data Explorer via different metrics including CPU utilization level, ingestion latency, ingestion result and query duration.
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context, and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure Data Explorer toolbox gives you an end-to-end solution for data ingestion, query, visualization and management. Download New Relic Azure Data Explorer monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Data Explorer service.
```
Comment (seemantk) [inline quickstarts/azure/azure-data-explorer/config.yml:19]:
```suggestion
  Monitoring Azure Data Explorer toolbox gives you an end-to-end solution for data ingestion, query, visualization and management. Install the New Relic Azure Data Explorer monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Data Explorer service.
```

File: quickstarts/azure/azure-data-lake-analytic/config.yml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+  ### Why monitor Azure Data Lake Analytics with New Relic?
+
+  [New Relic Azure Data Lake Analytics](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-data-lake-analytic-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure Data Lake Analytics via different metrics including A standard dashboard that tracks key indicators like number of jobs stage, AU time for jobs ended in success, failure and cancelled. 
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context, and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure Data Lake Analytics is critical to track the performance through key metrics. Download New Relic Azure Data Lake Analytics Service monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Data Lake Analytics service.
```
Comment (seemantk) [inline quickstarts/azure/azure-data-lake-analytic/config.yml:19]:
```suggestion
  Monitoring Azure Data Lake Analytics is critical to tracking the performance through key metrics. Install the New Relic Azure Data Lake Analytics Service monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Data Lake Analytics service.
```

File: quickstarts/azure/azure-data-lake-storage/config.yml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+ 
+  ### Why monitor Azure Data Lake Storage with New Relic?
+
+  [New Relic Azure Data Lake Storage](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-data-lake-storage-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure Data Lake Storage via different metrics including data read, data written, read requests, total storage and written requests. 
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context, and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure Data Lake Storage is critical to track the performance through key metrics. Download New Relic Azure Data Lake Storage monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Data Lake Storage service.
```
Comment (seemantk) [inline quickstarts/azure/azure-data-lake-storage/config.yml:19]:
```suggestion
  Monitoring Azure Data Lake Storage is critical to tracking the performance through key metrics. Install the New Relic Azure Data Lake Storage monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Data Lake Storage service.
```

File: quickstarts/azure/azure-device-provisioning-service/config.yml [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+  ### Why monitor Azure Device Provisioning Service with New Relic?
+
+  [New Relic Azure Device Provisioning Service](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-device-provisioning-service-integration/) monitoring quickstart empowers you to track the performance of Azure Device Provisioning Service via different metrics including account attestation attempts, device assignments and registration attempts.
+
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context and get valuable insights.
+
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure Device Provisioning Service is critical to track the performance through key metrics. Download New Relic Azure Device Provisioning Service monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Device Provisioning Service.
```
Comment (seemantk) [inline quickstarts/azure/azure-device-provisioning-service/config.yml:19]:
```suggestion
  Monitoring Azure Device Provisioning Service is critical to tracking the performance through key metrics. Install the New Relic Azure Device Provisioning Service monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Device Provisioning Service.
```

File: quickstarts/azure/azure-hdinsight/config.yml [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+  ### Why monitor Azure HDInsight with New Relic?
+
+  [New Relic Azure HDInsight](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-hdinsight-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure HDInsight via different metrics including categorized gateway requests, pending memory and pending CPU.
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context, and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure HDInsight is critical to track the performance through key metrics. Download New Relic Azure HDInsight monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure HDInsight service.
```
Comment (seemantk) [inline quickstarts/azure/azure-hdinsight/config.yml:19]:
```suggestion
  Monitoring Azure HDInsight is critical to tracking the performance through key metrics. Install the New Relic Azure HDInsight monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure HDInsight service.
```

File: quickstarts/azure/azure-integration-service-environment/config.yml [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+  ### Why monitor Azure Integration Service Environment with New Relic?
+
+  [New Relic Azure Integration Service Environment](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-integration-service-environment-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure Integration Service Environment via different metrics including actions completed, action latency, actions failed and connector memory usage. 
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure Integration Service Environment is critical to track the performance through key metrics. Download New Relic Azure Integration Service Environment monitoring quickstart to  get a pre-built dashboard tailored to monitor your Azure Integration Service Environment.
```
Comment (seemantk) [inline quickstarts/azure/azure-integration-service-environment/config.yml:19]:
```suggestion
  Monitoring Azure Integration Service Environment is critical to tracking the performance through key metrics. Install the New Relic Azure Integration Service Environment monitoring quickstart to  get a pre-built dashboard tailored to monitor your Azure Integration Service Environment.
```

File: quickstarts/azure/azure-iot-hub/config.yml [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+
+  ### Why monitor Azure IoT Hub with New Relic?
+
+  [New Relic Azure Iothub](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-iot-hub-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure IoT Hub via different metrics including jobs completed, jobs failed, D2C endpoints latency and more. 
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure IoT Hub is critical to track the performance through key metrics. Download New Relic Azure IoT Hub monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure IoT Hub service.
```
Comment (seemantk) [inline quickstarts/azure/azure-iot-hub/config.yml:19]:
```suggestion
  Monitoring Azure IoT Hub is critical to tracking the performance through key metrics. Install the New Relic Azure IoT Hub monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure IoT Hub service.
```

File: quickstarts/azure/azure-peering-service/config.yml [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+  
+  ### Why monitor Azure Peering Service with New Relic?
+
+  [New Relic Azure Peering Service](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-peering-service-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure Peering Service via round trip time. 
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure Peering Service is critical to track the performance through key metrics. Download New Relic Azure Peering Service monitoring quickstart to  get a pre-built dashboard tailored to monitor your Azure Peering Service.
```
Comment (seemantk) [inline quickstarts/azure/azure-peering-service/config.yml:19]:
```suggestion
  Monitoring Azure Peering Service is critical to tracking the performance through key metrics. Install the New Relic Azure Peering Service monitoring quickstart to  get a pre-built dashboard tailored to monitor your Azure Peering Service.
```

File: quickstarts/azure/azure-peering/config.yml [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+  ### Why monitor Azure Peering with New Relic?
+
+  [New Relic Azure Peering](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-peering-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure Peering via different metrics including egress traffic rate, flap counts, packet drop rate, session availability and more.
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context, and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure Peering is critical to track the performance through key metrics. Download New Relic Azure Peering monitoring quickstart to  get a pre-built dashboard tailored to monitor your Azure Peering.
```
Comment (seemantk) [inline quickstarts/azure/azure-peering/config.yml:20]:
```suggestion
  Monitoring Azure Peering is critical to tracking the performance through key metrics. Install the New Relic Azure Peering monitoring quickstart to  get a pre-built dashboard tailored to monitor your Azure Peering.
```

File: quickstarts/azure/azure-peering/config.yml [hunk 13]
```diff
@@ -0,0 +1,40 @@
+slug: azure-peering
+description: |-
+  ## What is Azure Peering?
+
+  Peering is the interconnection between Microsoft's global network (ASN AS8075) and your network for the purpose of exchanging internet traffic from/to Microsoft online services and Microsoft Azure Services. Carriers or Service Providers can request to connect with Microsoft at any of our Edge locations.
```
Comment (seemantk) [inline quickstarts/azure/azure-peering/config.yml:5]:
```suggestion
  Peering is the interconnection between Microsoft's global network (ASN AS8075) and your network for the purpose of exchanging internet traffic between Microsoft online services and Microsoft Azure Services. Carriers or Service Providers can request to connect at any of Microsoft's edge locations.
```

File: quickstarts/azure/azure-purview/config.yml [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+
+  ### Why monitor Microsoft Purview with New Relic?
+
+  [New Relic Microsoft Purview](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-purview-monitoring-integration/) monitoring quickstart empowers you to track the performance of Microsoft Purview via different metrics including scan failed count, scan cancelled count, scan completed count, data map capacity units count.
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Microsoft Purview is critical to track the performance through key metrics. Download New Relic Microsoft Purview monitoring quickstart to get a pre-built dashboard tailored to monitor your Microsoft Purview service.
```
Comment (seemantk) [inline quickstarts/azure/azure-purview/config.yml:19]:
```suggestion
  Monitoring Microsoft Purview is critical to tracking the performance through key metrics. Install the New Relic Microsoft Purview monitoring quickstart to get a pre-built dashboard tailored to monitor your Microsoft Purview service.
```

File: quickstarts/azure/azure-stream-analytic/config.yml [hunk 15]
```diff
@@ -0,0 +1,39 @@
+slug: azure-stream-analytic
```
Comment (seemantk) [inline quickstarts/azure/azure-stream-analytic/config.yml:1]: Should the slug actually be "azure-stream-analytics" to match the name of the service?

File: quickstarts/azure/azure-stream-analytic/config.yml [hunk 16]
```diff
@@ -0,0 +1,39 @@
...
+
+  ### Why monitor Azure Stream Analytics with New Relic?
+
+  [New Relic Azure Stream Analytics](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-stream-analytics-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure Stream Analytics via different metrics including failed requests, deserialization errors, late input events, resource utilization and more. 
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context, and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure Stream Analytics is critical to track the performance through key metrics. Download New Relic Azure Stream Analytics monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Stream Analytics service.
```
Comment (seemantk) [inline quickstarts/azure/azure-stream-analytic/config.yml:19]:
```suggestion
  Monitoring Azure Stream Analytics is critical to tracking the performance through key metrics. Install the New Relic Azure Stream Analytics monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Stream Analytics service.
```

File: quickstarts/azure/azure-synapse-analytic/config.yml [hunk 17]
```diff
@@ -0,0 +1,39 @@
+slug: azure-synapse-analytic
```
Comment (seemantk) [inline quickstarts/azure/azure-synapse-analytic/config.yml:1]: Should the slug be `azure-synapse-analytics` to match the name of the service?

File: quickstarts/azure/azure-synapse-analytic/config.yml [hunk 18]
```diff
@@ -0,0 +1,39 @@
...
+
+  ### Why monitor Azure Synapse Analytics with New Relic?
+
+  [New Relic Azure Synapse Analytics](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-synapse-analytics-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure Synapse Analytics via different metrics including account builtin sql pool data processed bytes, total sql streaming conversion errors, integration link processing latency in seconds, sql streaming resource utilization and more.
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context, and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure Synapse Analytics is critical to track the performance through key metrics. Download New Relic Azure Synapse Analytics monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Synapse Analytics service.
```
Comment (seemantk) [inline quickstarts/azure/azure-synapse-analytic/config.yml:19]:
```suggestion
  Monitoring Azure Synapse Analytics is critical to tracking the performance through key metrics. Install the New Relic Azure Synapse Analytics monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Synapse Analytics service.
```

File: quickstarts/azure/azure-time-series-insight/config.yml [hunk 19]
```diff
@@ -0,0 +1,40 @@
+slug: azure-time-series-insight
```
Comment (seemantk) [inline quickstarts/azure/azure-time-series-insight/config.yml:1]: Should the slug be `azure-time-series-insights` to match the name of the service?

File: quickstarts/azure/azure-time-series-insight/config.yml [hunk 20]
```diff
@@ -0,0 +1,40 @@
...
+
+  ### Why monitor Time Series Insights with New Relic?
+
+  [New Relic Azure Time Series Insights](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-time-series-insights-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure Time Series Insights via different metrics including ingress storage, ingress received bytes, ingress received messages and warm storage used properties. 
+  
+  Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context and get valuable insight.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure Time Series Insights Cloud is a fully visualization service that makes it easy to explore and analyze via key metrics. Download New Relic Azure Time Series Insight monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Time Series Insights service.
```
Comment (seemantk) [inline quickstarts/azure/azure-time-series-insight/config.yml:19]:
```suggestion
  Monitoring Azure Time Series Insights Cloud is a fully visualization service that makes it easy to explore and analyze via key metrics. Install the New Relic Azure Time Series Insight monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure Time Series Insights service.
```

**Review by seemantk — CHANGES_REQUESTED** [review]
Main points:

- slugs should match the service name
- quickstarts are not "downloaded"
- verb tenses should be consistent

**Comment by glennthomas1** [comment]
All issues resolved. Should be ready to go!

**Review by seemantk — APPROVED** [review]
LGTM

File: quickstarts/azure/azure-app-service-environment/config.yml [hunk 21]
```diff
@@ -0,0 +1,39 @@
...
+
+  App Service Environment represents the collection of physical resources for the App service. An App Service Environment can have multiple web apps. In other words, we can have multiple web apps in an App Service Environment.
+
+  ### New Relic Azure App Service Environment quickstart features
+
+  A standard dashboard that tracks key indicators like bytes received, bytes sent, cpu percentage, disk queue length, memory percentage. It runs custom queries and visualizes the data immediately.
+
+  ### Why monitor Azure App Service Environment with New Relic?
+
+  [New Relic Azure App Service Environment](//docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-app-service-environment-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure App Service Environment via different metrics including account bytes received, bytes sent, cpu percentage, disk queue length, memory percentage.
```
Comment (josephgregoryii) [inline quickstarts/azure/azure-app-service-environment/config.yml:13]:
```suggestion
  [New Relic Azure App Service Environment](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-app-service-environment-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure App Service Environment via different metrics including account bytes received, bytes sent, cpu percentage, disk queue length, memory percentage. 
```

**Review by josephgregoryii — COMMENTED** [review]
Just one last broken docs!

**Comment by glennthomas1** [comment]
Hopefully that resolves all issues and the merge and push to production can happen. Let me know...

**Comment by josephgregoryii** [comment]
@glennthomas1 that should be it! once the validations pass i'll send it through

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**